server/webhookNotifier.ts

```diff
@@ -107,10 +108,7 @@ const send = async (models, content: WebhookContent) => {
   });
   const chainOrCommwebhookUrls = [];
   chainOrCommWebhooks.forEach((wh) => {
-    // We currently only support slack webhooks
-    if (validURL(wh.url)) {
```

Comment:
For telegram web hook url, it doesn't match with `validURL` function. I think, we need to get @jnaviask 's review for this one. (Is it okay to remove `validURL` match part?)

Reply:
Will review tomorrow morning.

Reply:
Yeah this regex is insane, I don't think we need to use this -- or we should write a different validator specifically for TG webhooks.